model/roles_test.go

```diff
@@ -323,3 +323,160 @@ func TestGetRoleManifestDevPackageVersion(t *testing.T) {
 	differentExtraHash, _ := firstManifest.GetRoleManifestDevPackageVersion(firstManifest.Roles, "some string")
 	assert.NotEqual(firstHash, differentExtraHash, "role manifest hash should be dependent on extra string")
 }
+
+func TestLoadRoleManifestVariablesSortedBAD(t *testing.T) {
```

Comment:
The `BAD` looks odd. I think `OK` was just because it's an initialism, `Bad` or `Error` would be fine here.